frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -570,6 +570,11 @@ export const CRDCard: React.SFC<CRDCardProps> = (props) => {
     `/k8s/ns/${csv.metadata.namespace}/${ClusterServiceVersionModel.plural}/${
       csv.metadata.name
     }/${reference}/~new`;
+  if (crd.displayName.startsWith("[Internal]")) {
+    return (
+      <Card></Card>
```

Comment:
I doubt all will be showing since the PF Card component has some custom css being applied on card.
You can either return empty jsx elements to avoid this or return null.
Returning `null` would be good as its is suggested in [react docs](https://reactjs.org/docs/conditional-rendering.html#preventing-component-from-rendering)